### PR TITLE
Javascript/week3

### PR DESCRIPTION
**Pull request by narendra28051991**

### CODE REVIEW

File: javascript/javascript1/week3/js/main.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+let arrayOfNumbers = []
+
+function sumOfNumbers() {
+    for (let index = 0; index < 6; index++) {
+        let number = +prompt(`Enter number ${index + 1}: `);
+        arrayOfNumbers.push(number);
+    }
+
+    function sumOfEven() {
```
Comment (naresh699) [inline javascript/javascript1/week3/js/main.js:11]: can this function go outside and pass here with parameter (array) instead making nested function

File: javascript/javascript1/week3/js/main.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        let sum = 0;
+        for (let index = 0; index < arrayOfNumbers.length; index++) {
+            if (arrayOfNumbers[index] % 2 === 0) {
+                sum += arrayOfNumbers[index];
```
Comment (naresh699) [inline javascript/javascript1/week3/js/main.js:15]: check js terniary condition if it simplify your condition block

File: javascript/javascript1/week3/js/removeTheItem.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
```
Comment (naresh699) [inline javascript/javascript1/week3/js/removeTheItem.js:11]: ', ' is not needed in last index of array or obj

**Review by naresh699 — APPROVED** [review]
- code is ready to merge, once you have time to read about ternary condition.
- avoid nested funcion

File: javascript/javascript1/week3/js/main.js (hunk 1 above)
Comment (narendra28051991) [reply javascript/javascript1/week3/js/main.js:11]: Yes I will do that in the coming projects.
Infact I followed the lecture and want to implement it.

File: javascript/javascript1/week3/js/main.js (hunk 2 above)
Comment (narendra28051991) [reply javascript/javascript1/week3/js/main.js:15]: Thanks

File: javascript/javascript1/week3/js/removeTheItem.js (hunk 3 above)
Comment (narendra28051991) [reply javascript/javascript1/week3/js/removeTheItem.js:11]: I copied the array from the homework.md file. I will keep an eye next time.

I completely agree with your comments. I will improve my code.

**Pull request merged**